### ISSUE TITLE
Move series order to overview section

### ISSUE THREAD

**Issue by rami4625** [publishpress/publishpress-series#424]
I tried to change this option :
https://drive.google.com/file/d/1EBSH9TtqH3aV6o9YAghK3sQ6QRjAgMIS/view?usp=sharing

To Ascending but nothing changed in series-toc page,
Also i tried to change it to Series part and also nothing changed,

So i think the changes in this option not effect on front end.

**Comment by olatechpro** [comment]
@stevejburge , i think this is a UI issue, **'Order series**' which has options [Series part,  Order by date] as the name implies is for ordering series in series archive page and it works as designed. I think we gave impression of it been for series TOC because we put it directly after series TOC content since in real sense, we can't sort series TOC by series part, date etc.

**Comment by stevejburge** [comment]
@olatechpro OK, thanks

Do we need to add a new heading for this under the "Display" tab?

Does it fit under this new heading?
https://github.com/publishpress/publishpress-series/issues/405

**Comment by olatechpro** [comment]
Yes, it'll be better under Series Overview since the description is about taxonomy page

**Comment by stevejburge** [comment]
Original poster reports this isn't fixed
